### DISCUSSION TITLE
feat(misconf): Add support for AWS Identity Center Permission Set

### DISCUSSION THREAD

**Discussion by gantta** [aquasecurity/trivy#5877] [Ideas]
### Description

**Provider**
AWS

**Scanner**
misconfiguration

**Description**

## Abstract
When supplying security scan results to clients, we need the ability to show IAM Session Duration settings are defined in accordance with [NIST CPRT SP 800-53 Release 5.1.1 AC-12](https://csrc.nist.gov/projects/cprt/catalog#/cprt/framework/version/SP_800_53_5_1_1/home?element=AC-12)

## Proposal
I'm considering adding a new service similar to the following [merge request](https://github.com/aquasecurity/defsec/pull/1513)

## Links
[NIST CPRT SP 800-53 Release 5.1.1 AC-12](https://csrc.nist.gov/projects/cprt/catalog#/cprt/framework/version/SP_800_53_5_1_1/home?element=AC-12)
[CloudFormation Permission Set](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-sso-permissionset.html#cfn-sso-permissionset-sessionduration)
[Terraform Resource: aws_ssoadmin_permission_set](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssoadmin_permission_set#session_duration)

### Target

AWS

### Scanner

Misconfiguration

**Comment by gantta** [comment]
PR out here: https://github.com/aquasecurity/defsec/pull/1514

**Reply by simar7** [reply]
thanks converted into a feature issue https://github.com/aquasecurity/trivy/issues/5884
